include/cynq/accelerator.hpp

```diff
@@ -190,6 +274,37 @@ class IAccelerator {
                               elements * sizeof(T));
   }
 
+  /**
+   * @brief Read method (asynchronous)
+   * Performs a write operation to the accelerator through a register in
+   * an execution graph. It returns as soon as the instruction is scheduled
```

Comment:
this message is the same as line 125 but its spelling is inconsistent.

Reply:
I don't get this comment